### PR TITLE
SD-146 Touch devices selection model

### PR DESCRIPTION
**Pull request by awaitDzn**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts [hunk 1]
```diff
@@ -111,6 +107,5 @@ export class ContentPickerComponent implements OnInit, OnDestroy {
       this.subscription.unsubscribe();
     }
     // tslint:disable-next-line: deprecation
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts]: Remove tslint:disable-next-line: deprecation

File: src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts [hunk 2]
```diff
@@ -62,11 +62,7 @@ export class ContentPickerComponent implements OnInit, OnDestroy {
   }
 
   public ngOnInit() {
-    this.mobileQuery = this.media.matchMedia('(max-width: 600px)');
-    this.mobileQueryListener = () => this.changeDetectorRef.detectChanges();
     // tslint:disable-next-line: deprecation
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts]: Remove tslint:disable-next-line: deprecation

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts [hunk 3]
```diff
@@ -31,19 +32,21 @@ import { ContentDownloadDialogService } from '../dialog/components/content-downl
   ]
 })
 export class ContentBrowserComponent extends BaseBrowserComponent<Content> implements OnChanges {
-
   @Input()
   public channel: Channel | null = null;
 
+  public get deviceBreakpoint(): boolean {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts]: Add to BaseBrowserComponent as this needs to be reused for other browsers.

File: src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts (hunk 1 above)
Comment (awaitDzn) [reply src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts]: @neotek fixed

File: src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts (hunk 2 above)
Comment (awaitDzn) [reply src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts]: @neotek fixed

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts (hunk 3 above)
Comment (awaitDzn) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts]: @neotek done

**Pull request merged**